agent-install/agent-install.sh

```diff
@@ -18,8 +18,11 @@ SKIP_REGISTRATION=false
 CFG="agent-install.cfg"
 OVERWRITE=false
 HZN_NODE_POLICY=""
+AGENT_INSTALL_ZIP="agent-install-files.tar.gz"
```

Comment:
I dont see any code to pick up the pattern or node policy from the exchange. This is the mainline path for the batch install feature. Did I miss it?

Reply:
It is there now. The script runs 'hzn exchange node create' even when the node already exists in the exchange. Is this the expected behavior? As far as I can tell, that call doesn't change node policy or pattern info in the exchange.

Reply:
No. The problem is that the node will be precreated with a pattern (or node policy) AND the pattern or node policy will not be passed into the batch install script. The script is expected to retrieve that info from the exchange. I was looking at the code that assigns HZN_EXCHANGE_PATTERN and it seems to only assign it from input args.

Reply:
It was doing it in the registration method before, but I moved it up to the correct spot where HZN_EXCHANGE_PATTERN is set from input args.